.github/workflows/kata-deploy-test.yaml

```diff
@@ -0,0 +1,53 @@
+on: issue_comment
+name: test-kata-deploy
```

Comment:
On similar lines...Was just checking our existing release workflow and I see that it is still using the action from `https://github.com/kata-containers/packaging` : 
https://github.com/kata-containers/kata-containers/blob/ceebd06b64f086c21b4c5ee6fd7fb594559af5c1/.github/workflows/main.yaml#L297

We should fix that to use an action in this repo as well.

Reply:
That is for 1.x. release.yaml is for 2.x

Reply:
got it! The confusion while switching from 1.x to 2.x continues..:)